static/js/home.js

```diff
@@ -21,7 +21,8 @@ var homeJSLocal = {
   clearSelectAreaButton: undefined,
   detailViewState: false,
   waitingDialog: waitingDialog,
-  globalInit: false
+  globalInit: false, // this serves as the function of notifying the global initialization has been done.
+  on: false // It act as kind of mutual exclusion to allow only one thread to acquire the loading dialog box.
```

Comment:
The variable name on is too general, suggest renaming it to be more specific. such as mutexLoadingDialog